mds/apis/prv_api/vehicles.py

```diff
@@ -68,7 +68,7 @@ class DeviceSerializer(serializers.ModelSerializer):
         allow_null=True,
     )
     battery = serializers.FloatField(
-        source="latest_event.properties.telemetry.battery_pct",
+        source="latest_events[0].properties.telemetry.battery_pct",
         help_text="Percentage between 0 and 1",
         allow_null=True,
     )
```

Comment:
? Do you mean to just have on deviceSerializer that will be used when retrieving a device ? If so, I will do the change in another pull request, as it will be a pain to be merged, so i prefer to separate them